azurerm/internal/services/containers/kubernetes_cluster_node_pool_data_source.go

```diff
@@ -248,6 +253,14 @@ func dataSourceKubernetesClusterNodePoolRead(d *schema.ResourceData, meta interf
 		}
 		d.Set("priority", priority)
 
+		proximityPlacementGroupId := ""
+		if props.ProximityPlacementGroupID != nil {
+			proximityPlacementGroupId = string(*props.ProximityPlacementGroupID)
+		}
+		d.Set("proximity_placement_group_id", proximityPlacementGroupId)
+
+		d.Set("min_count", minCount)
```

Comment:
this is already set above: https://github.com/terraform-providers/terraform-provider-azurerm/pull/9195/files#diff-7ebfe4f8c2f575e50130c59761183244b759429db1dcce690bd9ef97c61cd7ecR219

```suggestion
```